packages/fxa-auth-server/fxa-oauth-server/lib/routes/introspect.js

```diff
@@ -32,13 +32,13 @@ module.exports = {
       iat: Joi.number().optional(),
       sub: Joi.string().optional(),
       iss: Joi.string().optional(),
-      jti: Joi.string().required(),
+      jti: Joi.string().optional(),
       'fxa-lastUsedAt': Joi.number().optional()
     })
   },
   handler: async function introspectEndpoint(req) {
     let token;
-    let tokenType = req.payload.token_type;
+    let tokenType = req.payload.token_type_hint;
```

Comment:
This would read a little clearer to me if the variable was named `tokenTypeHint` and `typeType` was set when the system determined the true token type.